spring-cloud-stream-dapr/spring-cloud-stream-binder-dapr-grpc/src/main/java/io/dapr/spring/cloud/stream/binder/messaging/DaprMessageConverter.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2022 The Dapr Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.dapr.spring.cloud.stream.binder.messaging;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.messaging.Message;
+import org.springframework.messaging.support.MessageBuilder;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.dapr.client.domain.PublishEventRequest;
+import io.dapr.v1.DaprAppCallbackProtos;
+
+/**
+ * Implement of DaprConverter.@see {@link DaprConverter}.
+ */
+public class DaprMessageConverter
```

Comment:
Should this be named JsonDaprMessageConverter since this is JSON specific?

Reply:
supported multiple ways in the latest commit so I keep the name as before